pkgs/by-name/rk/rkbin/package.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation {
+  pname = "rkbin";
+  version = "unstable-2024.02.22";
+
+  src = fetchFromGitHub {
+    owner = "rockchip-linux";
+    repo = "rkbin";
+    rev = "a2a0b89b6c8c612dca5ed9ed8a68db8a07f68bc0";
+    hash = "sha256-U/jeUsV7bhqMw3BljmO6SI07NCDAd/+sEp3dZnyXeeA=";
+  };
+
+  installPhase = ''
+    mkdir $out
+    mv bin doc $out/
+  '';
+
+  BL31_RK3568 = "bin/rk35/rk3568_ddr_1056MHz_v1.21.bin";
+  ROCKCHIP_TPL_RK3568 = "bin/rk35/rk3568_ddr_1056MHz_v1.21.bin";
+
+  ROCKCHIP_TPL_RK3588 = "bin/rk35/rk3588_ddr_lp4_2112MHz_lp5_2400MHz_v1.16.bin";
+
+  meta = with lib; {
+    description = "Rockchip proprietary bootloader blobs";
+    homepage = "https://github.com/rockchip-linux/rkbin";
+    license = licenses.unfree;
+    maintainers = with maintainers; [ thefossguy ];
```

Comment:
@thefossguy Could you add yourself to the list of maintainers in another PR targeting staging? Many thanks.

Reply:
Removed in #297511 temporarily.

Reply:
Oof, I am sorry for not seeing this until now.

Reply:
@wegank stupid question: I will create a new PR that adds me as the maintainer, but for that, should I still target the `staging` branch (which this PR targeted)?

Reply:
You can send it directly to staging, or send it to master and it'll get auto-merged into staging in a few hours.

Reply:
Done in #297643.